src/index.html

```diff
@@ -262,6 +262,125 @@ <h3 class="text-xl text-gray-50 font-semibold">
   </div>
 </section>
 
+<!-- Integrations -->
+<section id="integrations">
+  <div class="m-auto max-w-7xl px-4 py-12 md:pt-20 flex flex-col items-center">
+    <h2 class="text-gray-900 text-2xl md:text-3xl font-semibold text-center">
+      Integrations
+    </h2>
+    <span class="mt-3 md:max-w-sm text-gray-700 font-medium text-center">
+      It's easy to get start with Livebook, whether it be on the cloud or your local machine.
+    </span>
+    <a class="mt-8 max-w-7xl button button-outlined-gray whitespace-nowrap" href="#integrations">Browse Integrations</a>
```

Comment:
`max-w-7xl` is unused:

```suggestion
    <a class="mt-8 button button-outlined-gray whitespace-nowrap" href="#integrations">Browse Integrations</a>
```